### PR TITLE
Socint 146 revised ua cupdate event structure

### PR DESCRIPTION
**Pull request by PeterBochel**
CR: https://collaborate2.ons.gov.uk/jira/browse/SOCINT-146

### CODE REVIEW

File: src/test/java/uk/gov/ons/ctp/integration/rhsvc/message/impl/UacEventReceiverImplUnit_Test.java [hunk 1]
```diff
@@ -50,8 +50,8 @@ private void prepareAndAcceptEvent(String qid, EventTopic topic) {
     // Construct UacEvent
     UacEventFixture = new UacEvent();
     UacPayload uacPayloadFixture = UacEventFixture.getPayload();
-    uacFixture = uacPayloadFixture.getUac();
-    uacFixture.setQuestionnaireId(qid);
+    uacFixture = uacPayloadFixture.getUacUpdate();
+    uacFixture.setQid(qid);
```
Comment (rpcatling) [inline src/test/java/uk/gov/ons/ctp/integration/rhsvc/message/impl/UacEventReceiverImplUnit_Test.java:54]: should we be setting and testing more of the new attributes?

**Review by rpcatling — APPROVED** [review]
Some minor comments but not sufficient to prevent approving

File: docker/README.md [hunk 2]
```diff
@@ -118,7 +118,7 @@ and use Eclipse/IntelliJ to your current version.
 
 ## Example commands to run everything locally
 
-If not using the RH Cucumber local profile, point at the emulator using environment variables:
+If not using the RH Cucumber local profile, point at the emulator using environment variables (which are probably best added to your ~/.bashrc file):
```
[the remark this answers is not in the document]
Comment (philwhiles) [reply docker/README.md:121]: There is no helping some people!

File: src/main/java/uk/gov/ons/ctp/integration/rhsvc/service/impl/UniqueAccessCodeServiceImpl.java [hunk 3]
```diff
@@ -109,6 +109,7 @@ private UniqueAccessCodeDTO createUniqueAccessCodeDTO(
     }
 
     uniqueAccessCodeDTO.setCaseStatus(caseStatus);
+    uniqueAccessCodeDTO.setWave(uac.getMetadata().getWave());
```
[the remark this answers is not in the document]
Comment (PeterBochel) [reply src/main/java/uk/gov/ons/ctp/integration/rhsvc/service/impl/UniqueAccessCodeServiceImpl.java:112]: Converted to bean mapper code

**Pull request merged**